gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -259,26 +259,22 @@ MM_VerboseHandlerOutput::handleInitializedRegion(J9HookInterface** hook, uintptr
 }
 
 void
-MM_VerboseHandlerOutput::outputInitializedStanza(MM_EnvironmentBase *env, MM_VerboseManager *manager, MM_VerboseBuffer *buffer)
+MM_VerboseHandlerOutput::outputInitializedStanza(MM_EnvironmentBase *env, MM_VerboseBuffer *buffer)
 {
-
-	MM_GCExtensionsBase* extensions = env->getExtensions();
-
-	UDATA numaNodes = extensions->_numaManager.getAffinityLeaderCount();
 	char tagTemplate[200];
 
 	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
-	manager->setInitializedTime(omrtime_hires_clock());
-	getTagTemplate(tagTemplate, sizeof(tagTemplate), manager->getIdAndIncrement(), omrtime_current_time_millis());
+	_manager->setInitializedTime(omrtime_hires_clock());
```

Comment:
This can be removed from here. `InitializedTime` should only be set once, during startup of GC and it's handled in line 374. This time not actually used by this stanza, its just a marker of when GC is initialized and used to calculate interval between 2 of the same stanzas, when other stanzas are printed and there is 1 stanza, so this time is used.

Reply:
You can Assert instead, Assert(_manager->getInitializedTime() != 0)